odoo/modules/migration.py

```diff
@@ -85,8 +85,8 @@ def get_scripts(path):
 
             self.migrations[pkg.name] = {
                 'module': get_scripts(get_resource_path(pkg.name, 'migrations')),
-                'module_upgrades': get_scripts(get_resource_path(pkg.name, 'upgrades')),
-                'upgrades': get_scripts(_get_upgrades_paths(pkg.name)),
+                'module_upgrades': get_scripts(get_resource_path(pkg.name, 'upgrade')),
```

Comment:
I'm not sure about this change.
I think `upgrades` still make sense for the directory inside the modules. It's the directory you put your `upgrades` in.

Reply:
It's should also be in par with the legacy `migrations` directory.

Reply:
Is it already in use?

Reply:
maybe...